plugin/trino-hive/src/test/java/io/trino/plugin/hive/optimizer/TestHivePlans.java

```diff
@@ -245,7 +245,7 @@ public void testSubsumePartitionPartWhenOtherFilterNotConvertibleToTupleDomain()
                                         exchange(LOCAL,
                                                 exchange(REMOTE, REPARTITION,
                                                         project(
-                                                                filter("R_INT_COL IN (2, 3, 4)",
+                                                                filter("R_INT_COL IN (2, 3, 4) AND R_INT_COL BETWEEN 2 AND 4",
```

Comment:
regression? same issue as in `io.trino.plugin.hive.optimizer.TestHivePlans#testSubsumePartitionPartOfAFilter`?

Reply:
same response as https://github.com/trinodb/trino/pull/10985#discussion_r802759070 ?
will add a comment too